crates/tauri/src/app.rs

```diff
@@ -2165,6 +2165,26 @@ tauri::Builder::default()
       },
     };
 
+    // The env var must be set before the Runtime is created so that GetAvailableBrowserVersionString picks it up.
+    #[cfg(windows)]
+    {
+      if let crate::utils::config::WebviewInstallMode::FixedRuntime { path } =
+        &manager.config.bundle.windows.webview_install_mode
+      {
+        if let Some(exe_dir) = std::env::current_exe()
```

Comment:
can we use `utils::platform::current_exe` instead?

Reply:
ah whoops